.gitignore

```diff
@@ -60,9 +60,11 @@
 /build/msw/vc*_mswud
 /build/msw/wx_vc8arm_*.vcproj
 /build/msw/wx_vc8arm.sln
-/build/msw/ipch
-/build/msw/wx_vc12.opensdf
-/build/msw/wx_vc12.sdf
+/build/msw/ipch/
+/build/msw/*.opensdf
+/build/msw/*.sdf
+/build/msw/*.VC.db
+/build/msw/*.VC.VC.opendb
```

Comment:
Tbh I just copied patterns from [here](https://github.com/github/gitignore/blob/master/VisualStudio.gitignore#L87). 
And I didn't mention MSVC 2017. 

Anyway yes, I can confirm this. 
![wx](https://cloud.githubusercontent.com/assets/9366725/22702873/79061e48-ed62-11e6-9afa-dd5336d9da8a.png)